src/handlers/payout/action.ts

```diff
@@ -12,19 +12,51 @@ import {
   savePermitToDB,
   wasIssueReopened,
   getAllIssueAssignEvents,
+  addCommentToIssue,
 } from "../../helpers";
-import { UserType, Payload, StateReason } from "../../types";
+import { UserType, Payload, StateReason, Comment, User, Incentives, Issue } from "../../types";
 import { shortenEthAddress } from "../../utils";
 import { bountyInfo } from "../wildcard";
 import Decimal from "decimal.js";
 import { GLOBAL_STRINGS } from "../../configs";
 import { isParentIssue } from "../pricing";
+import { RewardsResponse } from "../comment";
+import { isEmpty } from "lodash";
 
-export const handleIssueClosed = async () => {
+export interface IncentivesCalculationResult {
+  paymentToken: string;
+  rpc: string;
+  networkId: number;
+  privateKey: string;
+  paymentPermitMaxPrice: number;
+  baseMultiplier: number;
+  incentives: Incentives;
+  issueCreatorMultiplier: number;
+  recipient: string;
+  multiplier: number;
+  issue: Issue;
+  payload: Payload;
+  comments: Comment[];
+  issueDetailed: {
+    isBounty: boolean;
+    timelabel: string;
+    priorityLabel: string;
+    priceLabel: string;
+  };
+  assignee: User;
+  tokenSymbol: string;
+}
+
+/**
+ * Function to check payment conditions
+ */
+
```

Comment:
the docstrings seems stalled

Reply:
> the docstrings seems stalled

Fixed here https://github.com/ubiquity/ubiquibot/pull/714/commits/57acf2c43dd5f1cfe334ffc09d463f061f4cb968